seleniumJSEnv/src/main/scala/org/scalajs/jsenv/selenium/DefaultFileMaterializer.scala

```diff
@@ -4,11 +4,8 @@ import org.scalajs.core.tools.io.VirtualTextFile
 import org.scalajs.jsenv.VirtualFileMaterializer
 import java.net.URL
 
-/** Materializes virtual files in a temporary directory and links to them
-  * via file://
-  */
-object DefaultFileMaterializer extends FileMaterializer {
-
+/** Materializes virtual files in a temp directory (uses file:// schema). */
+private[selenium] object DefaultFileMaterializer extends FileMaterializer {
```

Comment:
Similarly, maybe rename this to `FileMaterializerInTempDir` or `TempDirectoryFileMaterializer`?